cuqi/distribution/_gaussian.py

```diff
@@ -1,31 +1,43 @@
+import warnings
+import numbers
 import numpy as np
+import numpy.linalg as nplinalg
+
+import scipy as sp
 import scipy.stats as sps
-from scipy.sparse import diags, identity, issparse, vstack
-from scipy.sparse import linalg as splinalg
-from scipy.linalg import eigh, eigvals, cholesky
+import scipy.sparse as spa
+import scipy.linalg as splinalg
+
+# from scipy.sparse import diags, identity, issparse, vstack
+# from scipy.sparse import linalg as splinalg
+# from scipy.linalg import eigh, eigvals, cholesky
```

Comment:
```suggestion
```

Thanks for removing these. You can delete the comments

Reply:
Done